osmomath/sigfig_round_test.go

```diff
@@ -77,16 +75,21 @@ func TestSigFigRound(t *testing.T) {
 	for i, tc := range testCases {
 		tc := tc
 		t.Run(tc.name, func(t *testing.T) {
-			var actualResult sdk.Dec
-			osmoassert.ConditionalPanic(t, tc.tenToSigFig.Equal(sdk.ZeroInt()), func() {
-				actualResult = SigFigRound(tc.decimal, tc.tenToSigFig)
+			expPanic := tc.tenToSigFig.IsZero()
+			f := func() {
+				actualResult := SigFigRound(tc.decimal, tc.tenToSigFig)
 				require.Equal(
 					t,
 					tc.expectedResult,
 					actualResult,
 					fmt.Sprintf("test %d failed: expected value & actual value should be equal", i),
 				)
-			})
+			}
+			if expPanic {
```

Comment:
Can we make an osmomath helper for this? Seems frequent enough to justify a helper

Reply:
good point, I'll just add it in export_test.go for now